src/status_im/signals/core.cljs

```diff
@@ -95,50 +96,54 @@
         ^js event-js (.-event data)
         type         (.-type data)]
     (case type
-      "node.login"              (profile.login/login-node-signal cofx (transforms/js->clj event-js))
-      "backup.performed"        {:db (assoc-in db
-                                      [:profile/profile :last-backup]
-                                      (.-lastBackup event-js))}
-      "envelope.sent"           (transport.message/update-envelopes-status cofx
-                                                                           (:ids
+      "node.login"                 (profile.login/login-node-signal cofx (transforms/js->clj event-js))
```

Comment:
probably beyond the scope of this pr but we should really start moving this code into status_im2 namespace